mc_build.sh

```diff
@@ -46,10 +48,13 @@ build(){
     local CONTAINER_DEPENDS=""
     local BUILD_ARGS=""
     local MC_HOST=""
-    local VENDOR=$(echo "$MACHINE"|cut -d'-' -f1)
 
     declare -a GUESTS
 
+    if [ -z "$VENDOR" ]; then
+	    local VENDOR=$(echo "$MACHINE"|cut -d'-' -f1)
```

Comment:
I dont really understand what this is doing - you take in an OPTARG but then overwrite it with something generated from MACHINE? am i missing something?

Reply:
I take an optional optarg. If it doesn't exist, I try to pull it from MACHINE